docs/index.bs

```diff
@@ -2958,6 +2974,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       : Input
       :: |request|, a [=/request=]
+      :: |crossOriginIsolatedCapability|, a boolean
```

Comment:
It isn't clear to me which client this isolation refers to. Maybe it should just be `useHighResPerformanceTimers` or something which makes the scope of its usage clear?